### ISSUE TITLE
Cmake Overrides Directory

### ISSUE THREAD

**Issue by Hoikas** [zrax/string_theory#2]
On Windows, when utilizing string_theory as a dependency, its scripts override any string_theory directory specification given in the cmake gui. The overridden value is the local build directory. This seems to be a major blocker to using string_theory in CWE master.

**Comment by zrax** [comment]
I'm not sure I understand...  Are you saying that you're adding string_theory as a subproject to CWE?  The normal scripts seem to work fine for me when it's external (i.e. the usual cmake / build / install).

**Comment by Hoikas** [comment]
I'm doing the normal cmake / build / install. Then, when I configure CWE, the string_theory directory is always overridden with the string_theory build directory, rather than its install location. 

https://www.dropbox.com/s/uudv2c3ztuccd7z/Screenshot%202016-06-26%2021.03.44.png?dl=0

**Comment by Hoikas** [comment]
I think I have figured out how to reproduce the issue.

I do not use the environment variable `CMAKE_PREFIX_PATH` because I maintain multiple build environments. I need to be able to use MSVC++2013 for Blender/Python/Korman and MSVC++2015 for CWE. So, I typically set the `CMAKE_INSTALL_PREFIX` define per-project. This is what the CWE compile instructions say to do as well. However, when I set the `CMAKE_PREFIX_PATH` environment variable to the string_theory install prefix `D:\Libraries\VC++2015\x86`, `string_theory_DIR` is correctly found to be `D:/Libraries/VC++2015/x86/lib/cmake/string_theory`.

TL;DR - delete CMAKE_PREFIX_PATH environment variable to reproduce.

**Comment by zrax** [comment]
FYI, `CMAKE_PREFIX_PATH` can also be set per-project (e.g. `cmake -DCMAKE_PREFIX_PATH=C:\Devel\myprefix` or set it in the GUI before configuring).  I do this all the time to switch between build environments, and not just on Windows.  However, I still can't explain why CMake insists on picking a clearly invalid build directory as its import location -- or how it's even guessing it to begin with, for that matter.
